tests/javascript_tests/test_color_view_controller.js

```diff
@@ -205,6 +205,34 @@ requirejs([
       equal(this.dv.markers[idx+1].material.color.getHexString(), '000000');
     });
 
+    test("Testing toJSON", function() {
+      var container = $('<div id="does-not-exist" style="height:11px; width:12px"></div>');
+      var controller = new ColorViewController(container, this.sharedDecompositionViewDict);
+      //change color on one point
+      var idx = 0;
+      plottables = [{idx:idx}];
+      ColorViewController.setPlottableAttributes(this.dv, '#00ff00', plottables);
+
+      var obs = controller.toJSON();
+      var exp = "{\"category\":\"SampleID\",\"colormap\":\"discrete-coloring-qiime\",\"continuous\":false,\"colors\":{\"PC.636\":\"#ff0000\",\"PC.635\":\"#0000ff\"}}";
```

Comment:
I was under the impression that the return values were objects not strings, any idea of why this test is passing?